downstream/test-JTBD/Configure/ref-definition-file.adoc

```diff
@@ -0,0 +1,46 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="con-definition-file-breakdown"]
+
+= Breakdown of definition file content
+
+[role="_abstract"]
+You must provide a definition file to build {ExecEnvName} with {Builder}, because it specifies the content that is included in the {ExecEnvNameSing} container image.
+
+The following sections break down the different parts of a definition file.
+
+
+:_mod-docs-content-type: REFERENCE
```

Comment:
There's a mod-docs content type twice in this topic. Is it a reference or a concept? Either way, remove the extra. Or, is this supposed to be two files?